pegasus/sites.v3/hourofcode.com/public/learn/robotics.haml

```diff
@@ -0,0 +1,66 @@
+---
+title: Learn
+layout: wide_index
+social:
+  "og:description": "Anyone can learn the basic concepts of Computer Science. Choose an intro activity to start learning."
+  "og:image": "http://code.org/images/fit-520/codehoc3.jpg"
+  "og:image:width": 520
+  "og:image:height": 391
+  "og:video": ""
+  "og:video:width": ""
+  "og:video:height": ""
+  "og:video:type": ""
+---
+
+-# hourofcode.com uses an early version of Bootstrap 3 which didn't have this wider dialog.
+:css
+  @media (min-width: 992px)
+  {
+    .modal-lg {
+        width: 900px;
```

Comment:
same comment about close brace. im surprised you dont get errors about this when you load the page?